src/c.jl

```diff
@@ -1042,3 +1042,84 @@ function c2ixys(x, y, s)
     return permutedims(r)
 end
 
+"""
+    erfa_cp(p)
+
+Copy a p-vector.
+
+!!! warning "Deprecated"
+    Use `Base.copy` instead.
+
+### Given ###
+
+- `p`: p-vector to be copied
+
+### Returned ###
+
+- `c`: copy
+"""
+erfa_cp
```

Comment:
I do not want to pirate `Base.cp(::Array{Float64})`. Any better ideas?

Reply:
No :disappointed: